style.css

```diff
@@ -0,0 +1,992 @@
+body,
+h1,
+h2,
+h3,
+h4,
+p { margin: 0; }
+
+body {
+	min-height: 100vh;
+	line-height: 1.5;
+}
+
+img {
+	max-width: 100%;
+	display: block;
+}
+
+a { text-decoration: none; }
+
+/* Desktop */
+
+/* Шапка */
+.header,
+.header__menu,
+.header__wrapper_item {
+	display: flex;
+	justify-content: space-between;
+}
+
+.header {
+	align-items: center;
+	height: 104px;
+	margin: 0 auto;
+	margin-bottom: 30px;
+}
+
+.header__wrapper_logo {
+	display: flex;
+	justify-content: center;
+	flex: .21;
+}
+
+.header__menu { flex: .668; }
+
+.header__wrapper_item { flex: .486; }
+
+.header__link,
+.button__item_link { color: #000; }
+
+.header__wrapper {
+	display: flex;
+	justify-content: flex-start;
+	flex: .294;
+}
+
+.header__wrapper_link { margin-right: 30px; }
+
+.burger:hover { cursor: pointer; }
+
+.burger__icon {
+	display: block;
+	width: 25px;
+	height: 2px;
+	margin-bottom: 5px;
+	background-color: #e51937;
+	border-radius: 0 0 5px 5px;
+}
+
+.header__link:hover {
+	border-bottom: 2px solid #e51937;
+	border-radius: 2.5px;
+	color: #e51937;
+}
+
+/* Первый экран */
+.section-main {
+	margin-bottom: 34px;
+	background: url(./Project/tesla-wordmark-red_big.svg) no-repeat;
+	background-position: top;
+	background-size: 95%;
+}
+
+.section-main__title,
+.section-main__subtitle { text-align: center; }
+
+.section-main__title { height: 101px; }
+
+.section-main__subtitle,
+.section-main__img { margin-bottom: 26px; }
+
+.section-main__modelS-logo { width: 95%; }
+
+.car-logo,
+.section-main__wrapper { margin: auto; }
+
+.section-main__wrapper,
+.section-main__item_info,
+.section-main__item_btns,
+.character,
+.button_main,
+.button__item_alt,
+.button__item_big-order { display: flex; }
+
+.section-main__wrapper,
+.section-main__item_info,
+.section-main__item_btns { justify-content: space-between; }
+
+.section-main__wrapper {
+	width: 53%;
+	margin-bottom: 32px;
+}
+
+.section-main__item_info { flex: .48; }
+
+.character_right { padding-right: 5px; }
+
+.character__item_icon,
+.btn__item_icon { margin-right: 10px; }
+
+.section-main__item_btns,
+.btn__item_order,
+.button__item_alt,
+.button__item_big-order { align-items: center; }
+
+.section-main__item_btns { flex: .4; }
+
+.button__item_alt-right { flex: .83; }
+
+.button__item_big-order { height: 38px; }
+
+.btn__item_order,
+.section-main__item_control-down {
+	display: flex;
+	justify-content: center;
+}
+
+.btn__item_order {
+	width: 100%;
+	height: inherit;
+	border: 2px solid #e51937;
+	color: #e51937;
+}
+
+.button__item_link,
+.button__item_link-info { text-transform: uppercase; }
+
+/* Второй экран */
+.section-desc,
+.text__wrap_character,
+.text__item,
+.text__item_small-wifi { display: flex; }
+
+.section-desc,
+.button { margin: auto; }
+
+.section-desc {
+	width: 95%;
+	height: 900px;
+}
+
+.section-up {
+	background: url(./Project/item_interior.svg) no-repeat;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)